sql/moz-fx-data-shared-prod/google_ads_derived/campaign_conversions_by_date_v1/query.sql

```diff
@@ -0,0 +1,28 @@
+WITH campaigns AS (
+  SELECT
+    CAST(start_date AS DATE) AS start_date,
+    CAST(end_date AS DATE) AS end_date,
+    id AS campaign_id,
+    name,
+  FROM
+    `moz-fx-data-bq-fivetran`.google_ads.campaign_history
+)
+SELECT
+  date,
+  campaigns.name,
+  campaigns.campaign_id,
+  SUM(biddable_app_install_conversions) AS installs,
+  SUM(biddable_app_post_install_conversions) AS conversions,
+FROM
+  `moz-fx-data-bq-fivetran`.google_ads.campaign_conversions_by_date
+JOIN
```

Comment:
I confirmed that this join doesn't create any dupes in the output (grain is still date-campaign)